src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Api.Enduser/Controllers/ClientDelegationController.cs

```diff
@@ -0,0 +1,118 @@
+using System.ComponentModel.DataAnnotations;
+using System.Net.Mime;
+using Altinn.AccessManagement.Api.Enduser.Models;
+using Altinn.AccessManagement.Core.Constants;
+using Altinn.AccessManagement.Core.Models;
+using Altinn.AccessMgmt.Core;
+using Altinn.AccessMgmt.Core.Services;
+using Altinn.AccessMgmt.PersistenceEF.Audit;
+using Altinn.AccessMgmt.PersistenceEF.Utils;
+using Altinn.Authorization.Api.Contracts.AccessManagement;
+using Altinn.Authorization.ProblemDetails;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.FeatureManagement.Mvc;
+
+namespace Altinn.AccessManagement.Api.Enduser.Controllers;
+
+[ApiController]
+[Route("accessmanagement/api/v1/enduser/clientdelegations")]
+[FeatureGate(AccessMgmtFeatureFlags.EnduserControllerClientDelegation)]
+[Authorize(Policy = AuthzConstants.SCOPE_PORTAL_ENDUSER)]
+public class ClientDelegationController(IClientDelegationService clientDelegationService) : ControllerBase
+{
+    [HttpGet("clients")]
+    [Authorize(Policy = AuthzConstants.SCOPE_ENDUSER_CLIENTDELEGATION_READ)]
+    [ProducesResponseType<PaginatedResult<ConnectionDto>>(StatusCodes.Status200OK, MediaTypeNames.Application.Json)]
```

Comment:
ConnectionDto har vel ikke nødvendig respons modell for Access som skal inneholder hvilke pakker man har knytt til klient-forholdet
```
    "access": [
      {
        "role": "forretningsforer",
        "packages": [
          "forretningsforer-eiendom"
        ]
      },
      {
        "role": "regnskapsforer",
        "packages": [
          "regnskapsforer-med-signeringsrettighet",
          "regnskapsforer-uten-signeringsrettighet",
          "regnskapsforer-lonn"
        ]
      }
    ]
  },
```